server/__init__.py

```diff
@@ -88,6 +88,22 @@ def register_routes_custom_dc(app):
   pass
 
 
+def register_routes_stanford_dc(app):
+  # Install blueprints specific to Stanford DC
+  from routes import (disasters)
+  from routes.api import (disaster_api)
+  app.register_blueprint(disasters.bp)
+  app.register_blueprint(disaster_api.bp)
+
+  # load disaster dashboard data from GCS
+  if os.environ.get('FLASK_ENV') in [
```

Comment:
nit: we don't need this if statement because if we're registering stanford_dc_routes, we need to load the disaster dashboard data

Reply:
Fixed!